File: model/justificativa.php
<?php

class Justificativa{
    private $idJustificativa;
    private $idUsuario;
    private $idLocal;
    private $justificativa;
    private $dataHora;
    private $evidencia;

    function __construct($idJustificativa, $idUsuario, $idLocal, $justificativa, $dataHora, $evidencia){
        $this->setIdJustificativa($idJustificativa);
        $this->setIdUsuario($idUsuario);
        $this->setIdLocal($idLocal);
        $this->setJustificativa($justificativa);
        $this->setDataHora($dataHora);
        $this->setEvidencia($evidencia);
    }

    function setIdJustificativa($idJustificativa){
        $this->idJustificativa = $idJustificativa;
    }

    function setIdUsuario($idUsuario){
        $this->idUsuario = $idUsuario;
    }

    function setIdLocal($idLocal){
        $this->idLocal = $idLocal;
    }

    function setJustificativa($justificativa){
        $this->justificativa = $justificativa;
    }

    function setDataHora($dataHora){
        $this->dataHora = $dataHora;
    }

    function setEvidencia($evidencia){
        $this->evidencia = $evidencia;
    }

    function getIdJustificativa(){
        return $this->idJustificativa;
    }

    function getIdUsuario(){
        return $this->idUsuario;
    }

    function getIdLocal(){
        return $this->idLocal;
    }

    function getJustificativa(){
        return $this->justificativa;
    }

    function getDataHora(){
        return $this->dataHora;
    }

    function getEvidencia(){
        return $this->evidencia;
    }

    function __toString(){
        
        $data = (new DateTime($this->getDataHora()))->format('d/m/Y');
        $hora = (new DateTime($this->getDataHora()))->format('H:i:s');

        return
        "<br>ID da Justificativa: ".$this->getIdJustificativa().
        "<br>ID do Usuário: ".$this->getIdUsuario().
        "<br>ID Local: ".$this->getIdLocal().
        "<br>Justificativa: ".$this->getJustificativa().
        "<br>Data: ".$data.
        "<br>Hora: ".$hora."<br>";
    }
}

?>

File: aplicativo/app/src/main/java/com/example/echeckin/model/Justificativa.java
package com.example.echeckin.model;

import java.time.LocalDateTime;

public class Justificativa {

    private int idJustificativa;
    private int fkLocal;
    private int fkUsuario;
    private LocalDateTime dataHora;
    private String justificativa;

    public void Justificativa(int idJustificativa, int fkLocal, int fkUsuario, LocalDateTime dataHora, String justificativa){
        this.setIdJustificativa(idJustificativa);
        this.setFkLocal(fkLocal);
        this.setFkUsuario(fkUsuario);
        this.setDataHora(dataHora);
        this.setJustificativa(justificativa);
    }

    public void setIdJustificativa(int idJustificativa){
        this.idJustificativa = idJustificativa;
    }

    public void setFkLocal(int idLocal){
        this.fkLocal = fkLocal;
    }

    public void setFkUsuario(int fkUsuario){
        this.fkUsuario = fkUsuario;
    }

    public void setDataHora(LocalDateTime dataHora){
        this.dataHora = dataHora;
    }

    public void setJustificativa(String justificativa){
        this.justificativa = justificativa;
    }

    public int getIdJustificativa(){
        return this.idJustificativa;
    }

    public int getFkLocal(){
        return this.fkLocal;
    }

    public int getFkUsuario(){
        return this.fkUsuario;
    }

    public LocalDateTime getDataHora(){
        return this.dataHora;
    }

    public String getJustificativa(){
        return this.justificativa;
    }

}
